### PR TITLE
[New] Deploy react application (1k)

### PR DESCRIPTION
**Pull request by rsyracuse**
- Fixed javascript landing page
- Added React Landing Page
- Added React guide for CentOS 8, Debian 10, and Ubuntu 18.04

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 93b827da7db8274629d15fbf2adfbfa07b1bcf8d

https://deploy-preview-3279--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/development/react/how-to-deploy-a-react-app-on-debian-10/index.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+audiences: ["beginner"]
+concentrations: ["Web Applications"]
+languages: ["javascript"]
+---
+
+## What is React?
+
+[React](https://reactjs.org/) is a popular JavaScript library for building user interfaces. While React is often used as a frontend for more complex applications, it's also powerful enough to be used for full client-side applications on its own.
+
+Since a basic React app is static (it consists of compiled HTML, CSS, and JavaScript files), it is easy to deploy from a local computer to a Linode using [Rsync](/docs/tools-reference/tools/introduction-to-rsync/). This guide shows how to set up your Linode and local machine so that you can easily deploy your app whenever changes are made. See the [More Information](#more-information) section of this guide for
```
Comment (andystevensname) [inline docs/development/react/how-to-deploy-a-react-app-on-debian-10/index.md:28]: Finish last sentence of opening paragraph. Also, please bake in the distro name somewhere.

File: docs/development/react/how-to-deploy-a-react-app-on-centos-8/index.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+If you already have a React App that you would like to deploy to your Linode, you can skip this section. Otherwise, follow the steps in this section to create a basic React app using the [create-react-app](https://github.com/facebook/create-react-app#create-react-app--) tool.
+
+1. Use the Node Package Manager to create your React app.
+
+        npm init react-app ~/my-app
+
+### Create your Deployment Script
+
+1.  Navigate your app's directory. Replace `~/my-app` with the location of your React app's directory.
```
Comment (andystevensname) [inline docs/development/react/how-to-deploy-a-react-app-on-centos-8/index.md:149]: Navigate to

File: docs/development/react/how-to-deploy-a-react-app-on-debian-10/index.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+If you already have a React App that you would like to deploy to your Linode, you can skip this section. Otherwise, follow the steps in this section to create a basic React app using the [create-react-app](https://github.com/facebook/create-react-app#create-react-app--) tool.
+
+1. Use the Node Package Manager to create your React app.
+
+        npm init react-app ~/my-app
+
+### Create your Deployment Script
+
+1.  Navigate your app's directory. Replace `~/my-app` with the location of your React app's directory.
```
Comment (andystevensname) [inline docs/development/react/how-to-deploy-a-react-app-on-debian-10/index.md:151]: Navigate to

File: docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+If you already have a React App that you would like to deploy to your Linode, you can skip this section. Otherwise, follow the steps in this section to create a basic React app using the [create-react-app](https://github.com/facebook/create-react-app#create-react-app--) tool.
+
+1. Use the Node Package Manager to create your React app.
+
+        npm init react-app ~/my-app
+
+### Create your Deployment Script
+
+1.  Navigate your app's directory. Replace `~/my-app` with the location of your React app's directory.
```
Comment (andystevensname) [inline docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md:146]: Navigate to

File: docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+- '[Deploy a React App with Sass Using NGINX](http://zabana.me/notes/build-deploy-react-app-with-nginx.html)'
+audiences: ["beginner"]
+concentrations: ["Web Applications"]
+languages: ["javascript"]
+---
+## What is React?
+
+[React](https://reactjs.org/) is a popular JavaScript library for building user interfaces. While React is often used as a frontend for more complex applications, it's also powerful enough to be used for full client-side applications on its own.
+
+Since a basic React app is static (it consists of compiled HTML, CSS, and JavaScript files), it is easy to deploy from a local computer to a Linode using [Rsync](/docs/tools-reference/tools/introduction-to-rsync/). This guide shows how to set up your Linode and local machine so that you can easily deploy your app whenever changes are made. See the [More Information](#more-information) section of this guide for
```
Comment (andystevensname) [inline docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md:27]: Complete the thought in the last sentence, and please bake the distro name into this paragraph somewhere.

File: docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+1.  This guide will use `sudo` wherever possible. Complete the sections of our [Securing Your Server](/docs/security/securing-your-server/) to create a standard user account, harden SSH access, and remove unnecessary network services.
+
+1.  Install and configure a [web server](/docs/web-servers/) to host a website on your Linode. This guide's examples will use the Apache and NGINX web servers. Complete the steps in the [Installing Apache Web Server on Ubuntu 18.04 LTS](/docs/web-servers/apache/how-to-install-apache-web-server-ubuntu-18-04/) guide or the [Installing NGINX on Ubuntu 18.04 LTS](/docs/web-servers/nginx/how-to-install-nginx-ubuntu-18-04/) guide.
+
+1.  This guide assumes you already have a React app you'd like to deploy. If you don't have one, you can quickly bootstrap a project following the steps in the [Create an Example React App](#create-an-example-react-app)section of this guide. This step should be completed on your local system.
```
Comment (andystevensname) [inline docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md:37]: space between app and section.

File: docs/development/react/deploy-a-react-app-on-linode/index.md [hunk 7]
```diff
@@ -14,6 +14,9 @@ title: "Deploy a React Application on Linode"
 contributor:
   name: Phil Zona
   link: https://twitter.com/philzona
+aliases: ['development/javascript/deploy-a-react-app-on-linode/']
+deprecated: true
+deprecated_link: 'development/react/'
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/development/react/deploy-a-react-app-on-linode/index.md:19]: yes, it does

File: docs/development/react/how-to-deploy-a-react-app-on-centos-8/index.md [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+audiences: ["beginner"]
+concentrations: ["Web Applications"]
+languages: ["javascript"]
+---
+
+## What is React?
+
+[React](https://reactjs.org/) is a popular JavaScript library for building user interfaces. While React is often used as a frontend for more complex applications, it's also powerful enough to be used for full client-side applications on its own.
+
+Since a basic React app is static (it consists of compiled HTML, CSS, and JavaScript files), it is easy to deploy from a local computer to a Linode using [Rsync](/docs/tools-reference/tools/introduction-to-rsync/). This guide shows how to set up your Linode and local machine so that you can easily deploy your app whenever changes are made. See the [More Information](#more-information) section of this guide for
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/development/react/how-to-deploy-a-react-app-on-centos-8/index.md]: Last line should've been deleted. I've done that and added the distro to the opening paragraph.

File: docs/development/react/how-to-deploy-a-react-app-on-centos-8/index.md [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+
+1.  Familiarize yourself with our [Getting Started](/docs/getting-started/) guide and complete the steps for setting your [Linode's hostname](/docs/getting-started/#set-the-hostname) and [timezone](/docs/getting-started/#set-the-timezone).
+
+1.  This guide will use `sudo` wherever possible. Complete the sections of our [Securing Your Server](/docs/security/securing-your-server/) to create a standard user account, harden SSH access, and remove unnecessary network services.
+
+1.  Install and configure a [web server](/docs/web-servers/) to host a website on your Linode. This guide's examples will use the Apache and NGINX web servers. Complete the steps in the [Installing Apache Web Server on CentOS 8](/docs/web-servers/apache/how-to-install-apache-web-server-centos-8/) guide or the [Installing NGINX on CentOS 8](/docs/web-servers/nginx/how-to-install-nginx-centos-8/) guide.
+
+1.  This guide assumes you already have a React app you'd like to deploy. If you don't have one, you can quickly bootstrap a project following the steps in the [Create an Example React App](#create-an-example-react-app)section of this guide. This step should be completed on your local system.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/development/react/how-to-deploy-a-react-app-on-centos-8/index.md]: done

File: docs/development/react/how-to-deploy-a-react-app-on-debian-10/index.md (hunk 1 above)
Comment (leslitagordita) [reply docs/development/react/how-to-deploy-a-react-app-on-debian-10/index.md]: Last line should've been deleted. I've done that and added the distro to the opening paragraph.

File: docs/development/react/how-to-deploy-a-react-app-on-centos-8/index.md (hunk 2 above)
Comment (leslitagordita) [reply docs/development/react/how-to-deploy-a-react-app-on-centos-8/index.md]: done

File: docs/development/react/how-to-deploy-a-react-app-on-debian-10/index.md (hunk 3 above)
Comment (leslitagordita) [reply docs/development/react/how-to-deploy-a-react-app-on-debian-10/index.md]: done

File: docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md (hunk 4 above)
Comment (leslitagordita) [reply docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md]: done

File: docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md (hunk 5 above)
Comment (leslitagordita) [reply docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md]: Last line should've been deleted. I've done that and added the distro to the opening paragraph.

File: docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md (hunk 6 above)
Comment (leslitagordita) [reply docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md]: done

**Pull request merged**